caddyhttp/httpserver/replacer.go

```diff
@@ -226,6 +275,10 @@ func (r *replacer) Set(key, value string) {
 }
 
 const (
-	timeFormat     = "02/Jan/2006:15:04:05 -0700"
-	headerReplacer = "{>"
+	timeFormat        = "02/Jan/2006:15:04:05 -0700"
+	headerReplacer    = "{>"
+	headerContentType = "Content-Type"
+	contentTypeJSON   = "application/json"
+	contentTypeXML    = "application/xml"
+	maxLogBodySize    = 100 * 1000
```

Comment:
I think, technically, this should be `100 * 1024`... but I won't let what you have stop the merge. Just seems about 24 short of a nice, round number :wink: